src/Filter/ScssFilter.php

```diff
@@ -51,4 +54,130 @@ public function input($filename, $input)
         $return = $this->_runCmd($bin, '', array('PATH' => $this->_settings['path']));
         return $return;
     }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getDependencies(AssetTarget $target)
+    {
+        $children = [];
+        foreach ($target->files() as $file) {
+            $imports = CssUtils::extractImports($file->contents());
+            if (empty($imports)) {
+                continue;
+            }
+
+            $ext = $this->_settings['ext'];
+            $extLength = strlen($ext);
+
+            $deps = [];
+            foreach ($imports as $name) {
+                if (preg_match('/(.*)\/\*\*\/\*$/', $name, $matches)) {
```

Comment:
To be honest, I don't have experience in writing or using `phpunit` at all. I agree that we need tests for this, but I'm afraid I am not skillful enough to do that at the moment.
If you google about `sass_globbing` (it's a ruby gem), it uses that pattern for matching all the files recursively inside specified path.